docs/Paymaster/paymastermethods.md

```diff
@@ -5,39 +5,119 @@ sidebar_position: 3
 
 # Paymaster Methods
 
-This section will cover methods that can be called using the Paymaster Package.
-
-:::note
-
-When using these methods you will need to create a `userOp`. The [accounts methods](/account/methods) will help you in creating these for the paymaster methods below.
-
-:::
-
 Imports needed for these methods:
 
 ```ts
-import {
-  IHybridPaymaster,
-  PaymasterFeeQuote,
-  PaymasterMode,
-  SponsorUserOperationDto,
-} from "@biconomy/paymaster";
-```
-
-After setting up your smart account as mentioned above you can start using the Paymaster in this way:
+import { IHybridPaymaster, PaymasterFeeQuote,
+ PaymasterMode, SponsorUserOperationDto} from "@biconomy/paymaster";
 
-```ts
 const biconomyPaymaster =
   smartAccount.paymaster as IHybridPaymaster<SponsorUserOperationDto>;
 ```
 
-## getPaymasterAndData()
+After setting up your smart account as mentioned above you can start using the Paymaster.
 
-This method takes a `userOp` and optionally an object with information about sponsorship for the `userOp`. It returns a promise that resolves to a Paymaster and Data Response object. The data in this response can be used to update the userOp to include sponsorship data or data for paying gas in ERC20 tokens. There are two modes for using this function: `SPONSORED` and `ERC20`.
+## getPaymasterFeeQuotesOrData() 
+This method is particularly useful for `ERC20` mode and is used to get the fee quotes information for the ERC20 tokens.
+
+### Usage 
+
+```ts
+const feeQuotesResponse = await biconomyPaymaster.getPaymasterFeeQuotesOrData(
+  userOp,
+  {
+    mode: PaymasterMode.ERC20,
+    tokenList: ["0xdA5289FCAAF71d52A80A254dA614A192B693e975"],
+    preferredToken: "0xdA5289FCAAF71d52A80A254dA614A192B693e975",
+  },
+);
+```
+As per the code
+- We set mode to ERC20
```

Comment:
I think it needs more clarity on what response is received based on mode that is passed
`ERC20` and `SPONOSRED` mode both return different responses

Reply:
yes, in the usage section of this method, will clearly specify usage for both modes